apisix/utils/batch-processor-manager.lua

```diff
@@ -85,7 +86,20 @@ do
 end
 
 
-function _M:add_entry(conf, entry)
+local function total_processed_entries(self)
+    local processed_entries = 0
+    for _, log_buffer in pairs(self.buffers) do
```

Comment:
I am little worried it will cause bad performance

need a benchmark to confirm it

Reply:
According to my understanding, the number of `self.buffers` is proportional to the number of plugin configurations. For example, if kafka-logger is configured on 10 routes, there will be 10 `buffers`, so the quantity is controllable. 
@Revolyssup Am I right?